### PR TITLE
[DOC-3615] Reorg SMP getting started

### PR DESCRIPTION
**Pull request by brian-f-harness**
Reorg SMP getting started

For reviewers, a previews are available.

[Get started](https://64efc6e08b4b1512371f7c9b--harness-developer.netlify.app/docs/category/get-started-3)
[Overview](https://64efc6e08b4b1512371f7c9b--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/getting-started)
[Tutorials](https://64efc6e08b4b1512371f7c9b--harness-developer.netlify.app/docs/self-managed-enterprise-edition/introduction/smee-tutorials)

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/self-managed-enterprise-edition/introduction/getting-started.md [hunk 1]
```diff
@@ -1,19 +1,40 @@
...
+Review the following diagrams for your installation type.
+
+### Kubernetes cluster configuration
+
+Harness Self-Managed Enterprise Edition is installed in a Kubernetes cluster in the following configuration.
+
+![](./static/harness-self-managed-enterprise-edition-overview-00.png)To install Harness Self-Managed Enterprise Edition in a Kubernetes cluster, use the following instructions:
+
+1. For Self-Managed Enterprise Edition with Helm, go to [Install Harness Self-Managed Enterprise Edition Using Helm](../self-managed-helm-based-install/install-harness-self-managed-enterprise-edition-using-helm-ga.md).
+2. For Self-Managed Enterprise Edition with KOTS, go to [Install Self-Managed Enterprise Edition Using KOTS](../deploy-with-kots/kubernetes-cluster-on-prem-kubernetes-cluster-setup.md).
```
Comment (pratmit) [inline docs/self-managed-enterprise-edition/introduction/getting-started.md:31]: Remove this option as it is EOL.

File: docs/self-managed-enterprise-edition/introduction/getting-started.md [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
+redirect_from:
+  - /docs/self-managed-enterprise-edition/introduction/harness-self-managed-enterprise-edition-overview
+  - /docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition
 ---
 
-This topic provides the basics on how to create a Harness account and your first project. These are the first tasks that come after you install Harness Self-Managed Enterprise Edition.
+This topic provides an architecture overview, installation links, and the basics on how to create a Harness account and your first project. These are the first tasks that come after you install Harness Self-Managed Enterprise Edition.
 
-For links to information about using CI and CD pipelines, as well as the basics on Security Testing Orchestration (STO), go to [Next steps](/docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition#next-steps).
+For links to information about using CI and CD pipelines, as well as the basics on Security Testing Orchestration (STO), go to [Next steps](/docs/self-managed-enterprise-edition/introduction/getting-started#next-steps).
```
Comment (pratmit) [inline docs/self-managed-enterprise-edition/introduction/getting-started.md:17]: Remove this line

**Review by pratmit — APPROVED** [review]

**Review by deepakbolangady — APPROVED** [review]

File: docs/self-managed-enterprise-edition/introduction/getting-started.md [hunk 3]
```diff
@@ -1,19 +1,37 @@
...
 
-This topic provides the basics on how to create a Harness account and your first project. These are the first tasks that come after you install Harness Self-Managed Enterprise Edition.
+This topic provides an architecture overview, installation links, and the basics on how to create a Harness account and your first project. These are the first tasks that come after you install Harness Self-Managed Enterprise Edition.
 
-For links to information about using CI and CD pipelines, as well as the basics on Security Testing Orchestration (STO), go to [Next steps](/docs/self-managed-enterprise-edition/introduction/getting-started-with-self-managed-enterprise-edition#next-steps).
+To install Harness Self-Managed Enterprise Edition in a Kubernetes cluster with Helm, go to [Install Harness Self-Managed Enterprise Edition Using Helm](../self-managed-helm-based-install/install-harness-self-managed-enterprise-edition-using-helm-ga.md).
 
-### Create your Harness account
+## Architecture overview
+Harness Self-Managed Enterprise Edition can be installed in a Kubernetes cluster or on virtual machines.
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/introduction/getting-started.md:20]: Good suggestion, thanks. Updated.

File: docs/self-managed-enterprise-edition/introduction/getting-started.md [hunk 4]
```diff
@@ -1,19 +1,37 @@
...
+Review the following diagrams for your installation type.
+
+### Kubernetes cluster configuration
+
+Harness Self-Managed Enterprise Edition is installed in a Kubernetes cluster in the following configuration.
+
+![](./static/harness-self-managed-enterprise-edition-overview-00.png)
+
+**AWS architecture**
+
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/introduction/getting-started.md:31]: Yes, we should update here. Will note for a later PR.

File: docs/self-managed-enterprise-edition/introduction/smee-tutorials.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: Self-Managed Enterprise Edition tutorials
+description: Learn about Harness Self-Managed Enterprise Edition through step-by-step tutorials.
+sidebar_label: Tutorials
+sidebar_position: 10
+---
+
+In addition to the reference documentation, there are a several tutorials available.
```
[the remark this answers is not in the document]
Comment (brian-f-harness) [reply docs/self-managed-enterprise-edition/introduction/smee-tutorials.md:8]: Nice catch. Thanks.

**Pull request merged**